presto-tests/src/test/resources/resource_groups_client_tags_based_config.json

```diff
@@ -0,0 +1,66 @@
+{
+  "rootGroups": [
+    {
+      "name": "global",
+      "softMemoryLimit": "1MB",
+      "hardConcurrencyLimit": 100,
+      "maxQueued": 1000,
+      "softCpuLimit": "1h",
+      "hardCpuLimit": "1d",
+      "subGroups": [
+        {
+          "name": "a",
+          "softMemoryLimit": "2MB",
+          "hardConcurrencyLimit": 3,
+          "maxQueued": 4,
+          "subGroups": [
+            {
+              "name": "c",
+              "softMemoryLimit": "2MB",
+              "hardConcurrencyLimit": 3,
+              "maxQueued": 4
+            },
+            {
+              "name": "default",
+              "softMemoryLimit": "2MB",
+              "hardConcurrencyLimit": 3,
+              "maxQueued": 4
+            }
+
+          ]
+        },
+        {
+          "name": "b",
+          "softMemoryLimit": "2MB",
+          "hardConcurrencyLimit": 3,
+          "maxQueued": 4
+        },
+        {
+          "name": "other",
+          "softMemoryLimit": "2MB",
+          "hardConcurrencyLimit": 3,
+          "maxQueued": 4
+        }
+      ]
+    }
+  ],
+  "selectors": [
+    {
+      "clientTags" : "a, c",
```

Comment:
We still named it `clientTags` in session though it's even more obvious there that this is from client, right? Don't we want to be consistent here? I used a comma separated list because that's the from from session. But it does make more sense here to use a list. I don't have strong opinions either way so you guys can decide.

Reply:
This should be a list.